<ui version="4.0" >
 <class>frmTextDisplay</class>
 <widget class="QWidget" name="frmTextDisplay" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text Display - QSciTE</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>372</width>
     <height>405</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QCheckBox" name="cbWrapMode" >
      <property name="text" >
       <string>Wrap long lines</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblWrapMarkers" >
        <property name="text" >
         <string>Wrap indicators:</string>
        </property>
        <property name="buddy" >
         <cstring>cbxWrapMarkers</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxWrapMarkers" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Near text</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Near border</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbLineNos" >
        <property name="text" >
         <string>Line numbers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbIndentGuides" >
        <property name="text" >
         <string>Indentation guides</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblIndentSize" >
        <property name="text" >
         <string>Indent size:</string>
        </property>
        <property name="buddy" >
         <cstring>sbIndentSize</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbIndentSize" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAutoIndent" >
        <property name="text" >
         <string>Auto indent</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblUseTabs" >
        <property name="text" >
         <string>Indent using:</string>
        </property>
        <property name="buddy" >
         <cstring>cbxUseTabs</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxUseTabs" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>Spaces</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Tabs</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblLineEndings" >
        <property name="text" >
         <string>Line endings:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxEOLMode" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>CRLF (Windows)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>CR (Mac OS Classic)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>LF (Unix)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblLexer" >
        <property name="text" >
         <string>Lexer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxLexer" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QToolButton" name="btnFont" >
      <property name="toolButtonStyle" >
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbColor" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>96</height>
       </size>
      </property>
      <property name="title" >
       <string>Plain text color:</string>
      </property>
      <widget class="QWidget" name="layoutWidget_5" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>25</y>
         <width>341</width>
         <height>66</height>
        </rect>
       </property>
       <layout class="QGridLayout" >
        <property name="rightMargin" >
         <number>10</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="lblFore" >
          <property name="text" >
           <string>Foreground:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="sbFgR" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>R: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QSpinBox" name="sbFgG" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>G: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QSpinBox" name="sbFgB" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>B: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lblBack" >
          <property name="text" >
           <string>Background:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="sbBgR" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>R: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QSpinBox" name="sbBgG" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>G: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="sbBgB" >
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="prefix" >
           <string>B: </string>
          </property>
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="btnSaveSettings" >
        <property name="text" >
         <string>Save as default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset" >
        <property name="text" >
         <string>Reset to defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>frmTextDisplay</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>371</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>437</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
